cmd/main.go

```diff
@@ -19,13 +19,21 @@ type config struct {
 type argumentsError struct{}
 
 func (e *argumentsError) Error() string {
-	return "Usage: pair with <email>\n\nExample:\n\n  pair with 'Alice <alice@example.com>'\n"
+	return `Usage: 
+pair with <email>
+Example:
+  pair with 'Alice <alice@example.com>'
+
+pair stop
+Example:
+  pair stop
+`
 }
 
 func main() {
 	err := checkArgs(os.Args)
 	if err != nil {
-		fail(os.Stderr, err, 0)
+		os.Exit(fail(os.Stderr, err, 1))
```

Comment:
Yes, at this point the program should `os.Exit`. :+1: 

I would consider calling `pair` without arguments as a valid command, though. I am not strongly attached to that idea, but I think it is better not to change the API unless we have a strong case against exiting `0`. 

Any thougths @aakn?

Reply:
Yes, you're right. It should exit with status 0

Reply:
:+1:  I'll amend this, and `cherry-pick` the Golang container version change to #10 - I'm setting up Go Modules since I'm at it : )